src/test/ratings.spec.js

```diff
@@ -0,0 +1,51 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Testing Rating of articles', () => {
```

Comment:
Please apply the TTL's corrections in the implementation of your `description` and `it`. 
e.g. ```describe('HTTP-METHOD, route_path', () => {})```